filebeat/input/filestream/copytruncate_prospector.go

```diff
@@ -195,17 +189,6 @@ func (r rotatedFilestreams) addRotatedFile(original, rotated string, src loginp.
 	return -1
 }
 
-// addRotatedFile adds a new rotated file to the list and returns its index.
-// if a file is already added, the source is updated and the index is returned.
-func (r rotatedFilestreams) removeRotatedFile(original, rotated string) {
```

Comment:
For a SME: Perhaps this is a bug and this was meant to be used somewhere?

Reply:
@kvch thoughts on if this is a bug or not?

Reply:
I think it is a leftover from my work. I implemented a few different approaches for tracking rotated files. I probably forgot to clean up a few functions or kept it for when we will support rotating input files. IDK. But for now it can be removed.